eng/jacoco-test-coverage/pom.xml

```diff
@@ -104,12 +104,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
-      <version>4.3.2-beta.3</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+      <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos-encryption</artifactId>
-      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-encryption;current} -->
+      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-encryption;current} -->
```

Comment:
We don't need to increment this version - since cosmos-encryption is not releasing.

Reply:
Fixed